src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,34 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.9.0
+12 June, 2023
+
+### Deprecation Notice
+
+- Support for Kubernetes [1.23](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md) has been deprecated and will be removed in the next release.
+- Removed support for dedicated tls secrets for each ingress. 
+
+### Features
+
+- Kubernetes supported versions: [1.26](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.26.md), [1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md) and [1.24](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md).
+- Upgraded the Okteto CLI version to [2.16.1](https://github.com/okteto/okteto/releases/tag/2.16.1)
+- Telemetry job is now launched as soon as the helm installation is completed.
+- Now users have a new property `lastSeen` which is updated when the user interacts with Okteto. This property can be checked from the Admin dashboard, under the Users section.
+- Support to access Okteto on `127.0.0.1`
+- Upgrade ingress nginx to 4.5 
+
+### Improvements
+
+- Frontend now loads faster with improved nginx compression
+
+### Bugfixes
```

Comment:
@jpf-okteto Was the fix to prevent the error in the registry when there was an invalid manifest included in `1.9` or was it already on `1.8`?